srtcore/core.cpp

```diff
@@ -3422,17 +2740,19 @@ bool CUDT::checkApplyFilterConfig(const std::string &confstr)
     if (!PacketFilter::correctConfig(cfg))
         return false;
 
+    string thisconf = m_config.m_PacketFilterConfig.str();
+
     // Now parse your own string, if you have it.
-    if (m_OPT_PktFilterConfigString != "")
+    if (thisconf != "")
     {
         // - for rendezvous, both must be exactly the same, or only one side specified.
-        if (m_bRendezvous && m_OPT_PktFilterConfigString != confstr)
+        if (m_config.m_bRendezvous && thisconf != confstr)
```

Comment:
Not sure if this is right in the comment, but for rendezvous the fec config spec must be exactly identical.